### PR TITLE
Update CMakeLists.txt

### PR DESCRIPTION
**Pull request by SamayXD**
Updated Comments for better readability.

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -69,7 +69,7 @@ if (MSVC)
         /we4265 # 'class': class has virtual functions, but destructor is not virtual
         /we4388 # 'expression': signed/unsigned mismatch
         /we4389 # 'operator': signed/unsigned mismatch
-        /we4456 # Declaration of 'identifier' hides previous local declaration
+        /we4456 # Declaration of 'identifier' hides a previous local declaration
```
Comment (v1993) [inline src/CMakeLists.txt:72]: These are copied verbatim from warning messages and as such should be kept as is.

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -84,7 +84,7 @@ if (MSVC)
 
         /wd4100 # 'identifier': unreferenced formal parameter
         /wd4324 # 'struct_name': structure was padded due to __declspec(align())
-        /wd4201 # nonstandard extension used : nameless struct/union
+        /wd4201 # nonstandard extension used: nameless struct/union
```
Comment (v1993) [inline src/CMakeLists.txt:87]: Ditto.-

**Review by v1993 — CHANGES_REQUESTED** [review]

File: src/CMakeLists.txt (hunk 1 above)
Comment (SamayXD) [reply src/CMakeLists.txt:72]: Oh Sure.

File: src/CMakeLists.txt (hunk 2 above)
Comment (SamayXD) [reply src/CMakeLists.txt]: Is it fine now?

Comment (v1993) [reply src/CMakeLists.txt]: Yep.

**Review by liamwhite — APPROVED** [review]

**Pull request merged**